packages/eslint-plugin-gestalt/src/__fixtures__/no-spread-props/invalid-fixable-output.js

```diff
@@ -0,0 +1,21 @@
+import { Box, Box as RenamedBox } from 'gestalt';
+
+export default function TestElement(props) {
+  const onMouseEnterHandler = () => {};
+  const onMouseLeave = () => {};
+
+  const selectedWrapperProps = {
+    color: 'red',
+    onBlur: props.onBlur,
+    onClick: () => {},
+    onMouseEnter: onMouseEnterHandler,
+    onMouseLeave,
+    rounding: 2,
+  };
+
+  return (
+    <Box color="red" onBlur={props.onBlur} onClick={() => {}} onMouseEnter={onMouseEnterHandler} onMouseLeave={onMouseLeave} rounding={2} width={20}>
+      <RenamedBox color="red" onBlur={props.onBlur} onClick={() => {}} onMouseEnter={onMouseEnterHandler} onMouseLeave={onMouseLeave} rounding={2} width={20} />
```

Comment:
🤯 NICE!

Reply:
How would this handle something like
```js
<Component {...(condition ? propsA : propsB)} />
```
assuming both propsA and propsB are defined within the parent component? I suppose the more likely scenario is
```js
<Component {...(condition ? additionalProps : {})} />
```

Reply:
ATM this is a not-fixable scenario. 

Added ternary case to `/invalid-not-fixable-input.js`

Reply:
DONE